sim/druid/item_sets_pve_phase_6.go

```diff
@@ -237,11 +240,6 @@ func (druid *Druid) applyTAQGuardian4PBonus() {
 			spell.CD.FlatModifier -= 1500 * time.Millisecond
 		}
 	})
-	druid.OnInit(func(spell *core.Spell) {
-		if spell.SpellCode == SpellCode_DruidMangleBear {
-			spell.CD.FlatModifier -= 1500 * time.Millisecond
-		}
-	})
```

Comment:
For this bonus we need to create an aura and use the `HasAura` approach in order to not apply the bonus twice. You should be able to get rid of the `OnSpellRegistered`